docs/features/text-alignment.md

```diff
@@ -13,7 +13,11 @@ The {@link module:alignment/alignment~Alignment} feature enables support for tex
 
 ## Configuring alignment options
 
-It is possible to configure which alignment options are available in the editor by setting the {@link module:alignment/alignment~AlignmentConfig#options `alignment.options`} configuration option. You can choose from `'left'`, `'right'`, `'center'` and `'justify'`;  note that `'left'` should always be included.
+It is possible to configure which alignment options are available in the editor by setting the {@link module:alignment/alignment~AlignmentConfig#options `alignment.options`} configuration option. You can choose from `'left'`, `'right'`, `'center'` and `'justify'`.
+
+<info-box>
+	Note that `'left'` or `'right'` should always be included for <abbr title="left–to–right">LTR</abbr> and <abbr title="right-to-left">RTL</abbr> content, respectively. Learn more about {@link features/ui-language#setting-the-language-of-the-content configuring language of the editor content}.
```

Comment:
I had to thing trice to understand this. Could you write it in a more direct way – like:

> Note that the `'left'` option must be included if you set up the editor content to be LTR. Respectively, the `'right'` option should be included if you bla bla bla.

Reply:
```suggestion
	Note that the `'left'` option should always be included for the <abbr title="left–to–right">LTR</abbr> content. Similarly, the `'right'` option should always be included for the <abbr title="right-to-left">RTL</abbr> content. Learn more about {@link features/ui-language#setting-the-language-of-the-content configuring language of the editor content}.
```